ppr-api/src/ppr_api/models/registration.py

```diff
@@ -19,12 +19,12 @@
 
 from enum import Enum
 from http import HTTPStatus
-from datetime import date
 import json
 
 #from sqlalchemy import event
 
-from ppr_api.utils.datetime import format_ts, now_ts, now_ts_offset
+from ppr_api.utils.datetime import format_ts, now_ts, now_ts_offset,\
```

Comment:
i should've commented earlier on this.
my thoughts when making the datetime wrapper was to make it datetime, with more explicit support for timezones, mostly UTC
note sure these other helpers should be in the module wrapper.

Reply:
Moved helper datetime functions to models/utils.py.